src/Pluto.jl

```diff
@@ -26,6 +26,17 @@ const ENV_DEFAULTS = Dict(
         joinpath(preferred_dir, "") # must end with / or \
     end,
 )
+
+function default_env()
```

Comment:
If there's a convention that I should have used `"JULIA_PLUTO_*"` for all ENV names, then I'm okay with changing the names of all current pluto ENV variables. It's undocumented and probably not used by anyone

Reply:
I think the reason is we should provide environment variable starts with both `PLUTO` and `JULIA_PLUTO` just in case. Since this name `PLUTO` could be used by other things too. What do you think?

> Pluto already has a default ENV system that we can use, get_pl_env. This means that we only use the ENV name "PLUTO_PROJECT" (although see my later comment about the name).

I could update this too, since I'm also a bit concerned about the way it is implemented: current implementation will break the ability to create a distributable binary from Pluto using PackageCompiler, and I think at some point, we will need to do it at least for the CLI when things get more complicated.